auth-web/src/components/auth/common/ConfirmCancelButton.vue

```diff
@@ -62,27 +60,36 @@ export default class ConfirmCancelButton extends Vue {
   // for not to clear current org values [for account change , while clicking on cancel , current org has to stay]
   @Prop({ default: true }) clearCurrentOrg: boolean
 
-  private readonly resetAccountSetupProgress!: () => Promise<void>
+  @OrgModule.Action('setCurrentOrganizationFromUserAccountSettings') private setCurrentOrganizationFromUserAccountSettings!: () => Promise<void>
+  @OrgModule.Action('resetAccountSetupProgress') private resetAccountSetupProgress!: () => Promise<void>
 
   $refs: {
       confirmCancelDialog: ModalDialog
   }
 
-  private confirmDialogResponse (response) {
+  private async confirmDialogResponse (response) {
+    this.$refs.confirmCancelDialog.close()
```

Comment:
close need to be after respose?

Reply:
removed await so that the modal closes and the event runs in background